napari/layers/_base_layer/model.py

```diff
@@ -249,6 +259,7 @@ def freeze_refresh(self):
         yield
         self._freeze = False
 
+
```

Comment:
There should be only one blank line separating functions inside a class.

Reply:
ok ok...